cylc/flow/task_events_mgr.py

```diff
@@ -252,19 +252,27 @@ def check_job_time(self, itask, now):
         else:
             return can_poll
 
-    def get_host_conf(self, itask, key, default=None, skey="remote"):
-        """Return a host setting from suite then global configuration."""
+    def _get_remote_conf(self, itask, key):
+        """Get deprecated "[remote]" items that default to platforms."""
         overrides = self.broadcast_mgr.get_broadcast(itask.identity)
-        if skey in overrides and overrides[skey].get(key) is not None:
-            ret = overrides[skey][key]
-        elif itask.tdef.rtconfig[skey].get(key) not in (None, []):
-            ret = itask.tdef.rtconfig[skey][key]
-        else:
-            try:
-                ret = itask.platform[key]
-            except (KeyError, ItemNotFoundError):
```

Comment:
There's no way to get `ItemNotFoundError` here.  And `KeyError` can only occur if an illegal config item is requested.